eval.py

```diff
@@ -32,6 +32,8 @@ def parse_args():
                         help='number of input channels')
     parser.add_argument('--binf_map', type=str, default='misc/binf_map.csv',
                         help='Path to CSV with phonemes to binary features map')
+    parser.add_argument('--features_hparams_override', type=str, default='',
```

Comment:
Are you sure it is a good idea to adapt configuration that much for T2T *data format* support? Interaction between configuration options is already far from being straightforward and adding "override" flag that affects only parameters that were used by 3rd party framework to collect dataset seems to be an overcomplication. As the very minimum we should rename this to something like `--t2t_hparams_override`.

Reply:
Renamed it.
We have added not only _optional_ T2T data format, but also faster features computation )
In order to utilize T2T features problems, we can either follow their workflow by adding support for path of custom T2T problem. where custom parameters are defined, or simply pass them via `hparams` override string.

Reply:
Well, it can't be faster than pre-computed. :)

Reply:
Our calculation is rather slow. We can move this calculation workflow one day to our preprocessing script. BTW. `build_vocab` runs circa 8 minutes, with all features being calculated if T2T dataset is used.

Reply:
Hm... I ran slightly older version on TIMIT and it was fast: `3695phrase [00:01, 2685.81phrase/s] Max frames: 779, max symbols: 74`. It is constrained by IO, so probably moving data to SSD or increasing number of parallel threads should help.